js/package.json

```diff
@@ -48,6 +48,7 @@
     "lint": "npm run lint:css && npm run lint:js",
     "lint:cached": "npm run lint:css && npm run lint:js:cached",
     "lint:css": "stylelint ./src/**/*.css",
+    "lint:fix": "./node_modules/.bin/eslint --fix 'src/**/*.spec.js' && ./node_modules/.bin/eslint --fix 'src/**/*.js'",
```

Comment:
Since we are running via `npm run lint:fix` tou can get rid of the `./node_modules/.bin/` prefix